### PR TITLE
Add go modules & circleCI orb

### PR DESCRIPTION
**Pull request by georgi-denchew**
# Description

## What

Migrate to go modules and use the circleCI orb

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,58 +1,19 @@
...
 workflows:
-  version: 2
-  test-and-build-docker:
+  tests_and_docker:
     jobs:
-      - build
-      - dockerfile:
+      - ft-golang-ci/build-and-test:
+          name: build-and-test-project
+      - ft-golang-ci/docker-build:
```
Comment (ivan-p-nikolov) [inline .circleci/config.yml:11]: Shouldn't we add `dockerhub-shared` context

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module github.com/Financial-Times/annotations-monitoring-service
+
+go 1.14
```
Comment (ivan-p-nikolov) [inline go.mod:3]: I think you need to update your local golang version. The current one is 1.15. 
And It's fine to keep 1.14 in here.

File: README.md [hunk 3]
```diff
@@ -10,18 +10,15 @@ Service responsible for monitoring annotations publishes.
...
-        go build .
+        go build
 
 ## Running locally
 
 1. Run the tests and install the binary:
 
-        govendor sync
-        govendor test -v -race
+        go test -race ./...
```
Comment (mchompalova) [inline README.md:21]: We should add -mod=readonly here

**Review by mchompalova — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mchompalova) [reply .circleci/config.yml:11]: It's not necessary anymore.

**Review by mchompalova — DISMISSED** [review]

**Comment by ekaneva** [comment]
@georgi-denchew
Metadata marked this service as one that is going to be decommissioned soon in [the service list](https://docs.google.com/spreadsheets/d/1_r7biMVRIxUMEwAviQP4iAFbMTvQWbWGzdY2BNbLfgY/edit#gid=847291464), so I think it is better to close this PR?

**Review by martin-stanchev — APPROVED** [review]

**Review by dtvalk-ov — APPROVED** [review]

**Pull request merged**